testcases/others/regexp/reg4

```diff
@@ -0,0 +1 @@
+<(.|\n)+?>
```

Comment:
looks like only this file is in the `regexp` dir, three others are still in `regex`, did you forget to run `git add` ? :)

Reply:
My bad, I've fixed them. Aside, I'll be keep updating this. Thank you once again.